src/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,238 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React, { Component } from 'react';
+import { createStyledComponent } from '../styles';
+import { createThemedComponent, mapComponentThemes } from '../themes';
+import Popover, {
+  componentTheme as popoverComponentTheme
+} from '../Popover/Popover';
+
+type Props = {
+  /** Trigger for the Tooltip */
+  children: React$Node,
+  /** Content of the Tooltip */
+  content: string,
+  /** For use with uncontrolled components, in which the Tooltip is immediately upon initialization */
+  defaultIsOpen?: boolean,
+  /** @Private Tooltips cannot be disabled and this prop will be stripped */
+  disabled?: boolean,
```

Comment:
Something's odd here. Despite this comment, you have a spec for a disabled tooltip. Also, adding `disabled` to a Tooltip in the demo doesn't seem to do anything...

Reply:
Updated disabled prop description